### DISCUSSION TITLE
[AL_USDMaya] ProxyDrawOverride/ProxyShapeSelect removal

### DISCUSSION THREAD

**Discussion by fabal** [Autodesk/maya-usd#1879] [Announcement]
Hi there,
as we are working towards replacing AL_USDMaya with maya-usd in Animal Logic's pipeline and considering the size of our development team, we've decided to be more aggressive about the code base we support. The first "victims" are:
* the `proxy draw override` display (replaced a long time ago by the `VP2 proxy render delegate`)
* the `AL_usdmaya_ProxyShapeSelect` command (superseded by UFE selection)
We're hoping that this decision won't affect anyone but let us know if that's the case.

**Comment by kxl-adsk** [comment]
Definitely, something we (at Autodesk) are supporting.
